modules/videoio/test/test_video_io.cpp

```diff
@@ -69,6 +70,7 @@ class Videoio_Test_Base
     }
     void checkFrameSeek(int idx, VideoCapture & cap)
     {
+#ifndef HAVE_AVFOUNDATION
```

Comment:
This check should be replaced with dynamic check. See my other comment for details.